docs/running/cloud/kubernetes.rst

```diff
@@ -17,6 +17,8 @@ Preparing your Kubernetes environment
 
    To run Toil workflows on Kubernetes, you need to have a Kubernetes cluster set up. This will not be covered here, but there are many options available, and which one you choose will depend on which cloud ecosystem if any you use already, and on pricing. If you are just following along with the documentation, use ``minikube`` on your local machine.
 
+   Alternatively, if you are using the :ref:`Toil provisioner <installProvisioner>`, Toil can set up a Kubernetes cluster for you. Follow :ref:`this <ProvisioningWithKubernetes>` guide to get started with a Toil-managed Kubernetes cluster on AWS.
```

Comment:
```suggestion
   Alternatively, Toil can set up a Kubernetes cluster for you with the :ref:`Toil provisioner <installProvisioner>`. Follow :ref:`this <ProvisioningWithKubernetes>` guide to get started with a Toil-managed Kubernetes cluster on AWS.
```

I don't think using or not using the provisioner is like an overall state that needs to be coped with; they can go get the provisioner setup if they need it.

Reply:
Makes sense!